pkg/cri/store/image/image.go

```diff
@@ -85,6 +86,14 @@ func (s *Store) Update(ctx context.Context, ref string) error {
 			return fmt.Errorf("get image info from containerd: %w", err)
 		}
 	}
+	// parsing ref consistently for refCache, including adding `docker.io/library` prefix
```

Comment:
If this is for normalizing cache keys, how about doing that right before touching refCache?

Reply:
Move to `func update()` below then.